homeassistant/components/history.py

```diff
@@ -59,8 +59,8 @@ def get_significant_states(hass, start_time, end_time=None, entity_id=None,
     """
     timer_start = time.perf_counter()
     from homeassistant.components.recorder.models import States
-
-    entity_ids = (entity_id.lower(), ) if entity_id is not None else None
+    entity_ids = entity_id.lower().split(
```

Comment:
Readability is suffering, would prefer if we just make it a normal `if…else`

Reply:
We shouldn't split strings here, I think that it makes more sense to expect a list here. Argument parsing should happen before we get to this API.

Reply:
It's a public method that could be in use by custom components.
I'll change it to a list but we should mark it as breaking change in that case.

Reply:
We can also allow either.

```python
if not isinstance(entity_id, list):
    entity_id = [entity_id]
```